docs/content/en/docs/packages/emissary/addemissary.md

```diff
@@ -15,6 +15,10 @@ Be sure to refer to the [troubleshooting guide]({{< relref "../troubleshoot" >}}
    * While following this guide to install packages on a workload cluster, please make sure the `kubeconfig` is pointing to the management cluster that was used to create the workload cluster. The only exception is the `kubectl create namespace` command below, which should be run with `kubeconfig` pointing to the workload cluster.
    {{% /alert %}}
 
+{{% alert title="Note" color="primary" %}}
+Operators can also leverage Ingress support provided by Isovalent that uses the standard Kubernetes Ingress resource definition, with an ingressClassName of cilium. This can be used for path-based routing and for TLS termination. (https://docs.cilium.io/en/stable/network/servicemesh/ingress/)
```

Comment:
```suggestion
Operators can also leverage Ingress support provided by Cilium that uses the standard Kubernetes Ingress resource definition, with an ingressClassName of cilium. This can be used for path-based routing and for TLS termination. (https://docs.cilium.io/en/stable/network/servicemesh/ingress/)
```

Reply:
Is this definitely enabled on EKS Anywhere Cilium? I didn't think service mesh was a feature and I noticed the doc link is under a service mesh section.

Reply:
Hi @chrisdoherty4 for EKS-A ingress is called out here- https://anywhere.eks.amazonaws.com/docs/workloadmgmt/ingress/ 

From Cilium's perspective we actually club it under servicemesh as you rightly pointed out in that link. Please let us know if that is ok.